### PR TITLE
NAS-4708 Override axios transform with function that explicitly returns raw data per comment

### PR DESCRIPTION
**Pull request by provCyd**

### CODE REVIEW

**Comment by provCyd** [comment]
Pinged about this, will merge by EOD if no feedback is received.

File: src/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ export default class extends XHRBackend {
       Object.assign(
         {
           ajax: (url, { init } = {}, cb) => axios(url, Object.assign({
-            transformResponse: undefined, // i18next expects JSON, not an object
+            transformResponse: [(data) => data], // i18next expects JSON, not an object
```
Comment (provCyd) [inline src/index.js:11]: Passing an empty array may also work, but it seems better to explicitly return untransformed data and this is a tested fix.

**Comment by provCyd** [comment]
For what it's worth, after writing this fix I found a recommendation for the exact same implementation: https://stackoverflow.com/questions/41013082/disable-json-parsing-in-axios

**Review by provjordan — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (provCyd) [reply src/index.js:11]: should also be `data => data`, just fixed that up

**Pull request merged**